src_py/apiServer/transmitter.py

```diff
@@ -125,8 +125,8 @@ def predict(self):
             time.sleep(0.005)
             pass 
         
-        globe.expResults.remove0Tails()
-        globe.multiProcQueue.put(globe.expResults)
+        globe.experiment_flow_global.remove0Tails()
```

Comment:
remove0Tails? is it a typo? 
Why 0?

Reply:
Function created by Dor, by the name I assume it removes trailing "0" from experiment data?

Reply:
Ok let's keep it as it is